### PR TITLE
[python] Update files in ros2controlcli to use format strings

### LINKED ISSUE #355: use format strings for ros2controlcli
rather than using `.format()`, we can use `f'...'` in Python3.6

see https://www.python.org/dev/peps/pep-0498/

### PR DESCRIPTION
**Pull request by NovusEdge**
Fixes #355 

@bmagyar 
Please be sure to review the changes and let me know if something is wrong; or if more changes are to be made.

### CODE REVIEW

**Comment by destogl** [comment]
@NovusEdge This looks great thanks! Did you also test it?

**Comment by NovusEdge** [comment]
I'm afraid i didn't, do you mind running the tests and letting me know if there's any problem?

**Comment by destogl** [comment]
@NovusEdge I am guessing you are new in ros2_control and would like to learn more about the framework. Am I right?

If so, I propose to check out ros-controls/ros2_control_demos and test the call using rrbot demo hardware. Also, before doing anything further with this PR, you should check these errors with linters.

**Review by destogl — DISMISSED** [review]
As per comment. Also, check the errors with linters. You can also execute them locally using `ament_*` commands.

**Comment by Karsten1987** [comment]
@NovusEdge friendly ping. Are you still working on this?

**Comment by NovusEdge** [comment]
> @NovusEdge friendly ping. Are you still working on this?

I currently *can't* work on it. So, no, I'm not working on it.

File: ros2controlcli/ros2controlcli/api/__init__.py [hunk 1]
```diff
@@ -32,25 +32,24 @@ def service_caller(service_name, service_type, request):
         rclpy.init()
 
         node = rclpy.create_node(
-            'ros2controlcli_{}_requester'.format(
-                service_name.replace(
-                    '/', '')))
+            f"ros2controlcli_{ service_name.replace('/', '') }_requester"
```
Comment (bmagyar) [inline ros2controlcli/ros2controlcli/api/__init__.py:35]:
```suggestion
            f"ros2controlcli_{ service_name.replace('/', '') }_requester")
```

**Review by bmagyar — APPROVED** [review]

**Pull request merged**